### DISCUSSION TITLE
gh actions-importer update command is failing

### DISCUSSION THREAD

**Discussion by leandromsft** [github/gh-actions-importer#306] [Feedback]
Hi 

I'm trying to run this command **gh actions-importer update** but result the error bellow

Updating ghcr.io/actions-importer/cli:latest...
There was an error pulling the ghcr.io/actions-importer/cli:latest.
Error: Error response from daemon: Head "https://ghcr.io/v2/actions-importer/cli/manifests/latest": denied: denied

**Comment by homj** [comment]
Same for me. This makes the actions-importer completely unusable.

EDIT: @leandromsft in the end it was only because i was not authenticated to the ghcr.io package registry. Running `docker login ghcr.io` and logging in with your credentials should fix this

**Comment by leandromsft** [comment]
Thank you @homj very much for the workaround!

But even so I think the command "gh actions-importer update" should incorporate docker login or document that it is necessary to do docker login manually first

**Comment by hellolsun** [comment]
I got the following errors when running "gh actions-importer update" :

Updating ghcr.io/actions-importer/cli:latest...
There was an error pulling the ghcr.io/actions-importer/cli:latest.
Error: error pulling image configuration: download failed after attempts=6: Forbidden

After I ran "docker login ghcr.io" and got "Login Succeeded" message, I still got the same "Forbidden" message on " gh actions-importer update". Any thoughts?
